lib/cosmos/gui/qt.rb

```diff
@@ -91,27 +91,25 @@ module Cosmos
   def self.getColor(color_r, color_g = nil, color_b = nil)
     return color_r if (color_r.is_a? Qt::Color) || (color_r.is_a? Qt::Pen) || (color_r.is_a? Qt::LinearGradient)
 
-    color = nil
     key = color_r
-    key = key.to_i if key.is_a? Qt::Enum
+    # Check for a cached color right off the bat as an optimization
+    color = Cosmos::COLORS[key]
+    return color if color
```

Comment:
This reordering breaks red, green, blue paramters.  Can't just use red as the key